src/index.js

```diff
@@ -2,8 +2,31 @@ import './style.css';
 import Task from '../modules/task.js';
 
 const task = new Task();
-task.addList('Dance with Sanja', true, 0);
-task.addList('Dance with Toma', false, 1);
-task.addList('Dance with Waleed', true);
-task.addList('Dance with max', true, 3);
-// task.addList({description: 'Dance with Sanja', completed: true, index: 1});
+window.addEventListener('load', task.showLists());
```

Comment:
- [ ] Great job so far 👏🏻 however In this project, you should implement the CRUD (create, update, delete) methods Fistly you should have a new JavaScript file that will include: 
  -  A function for editing task descriptions make sure you implement it. 🥲 
  -  A function for deleting a task (remove an element from the array). (Note: Youcan also implement the deleting feature like [this](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/sneak_peek.md#:~:text=Your%20task%20is%20to%20build%20a%20website%20that%20displays%20a%20list%20that%20looks%20and%20behaves%20like%20the%20part%20of%20minimalist%20project%20capture%20in%20the%20video%20below.)).🥲 
  - Make sure new tasks have the property `completed` set to `false`  by default and the property `index` set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal to 5).🥲 
  - Don't forget, Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique.🥲 
  - And make sure, All changes to the To-Do List is saved in local storage. 🥲 
 

- [ ] Kindly implement all those new functionalities currently I can only create the book, I hope this is more clear now👍🏻 . If you have any questions please do not hesitate to tag me on @BenMukebo or send me a direct message on slack✌🏻

Reply:
I already Implemented them

Reply:
Okay let me check again